src/layouts/shared/page-header.tsx

```diff
@@ -100,22 +100,6 @@ function Navigation({
       >
         <ul>
           <li></li>
```

Comment:
These changes are good, this is more a general question that is related to this. Should we populate the header when the user is on the details page, or is it okay for the user to go back to the assistant/departure page then change view?